src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,30 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.dao.ManufacturerDaoImpl;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
+
```

Comment:
Add injector. See task description 
```suggestion
          private static final Injector injector = Injector.getInstance("YOUR_PACKAGE");

```